packages/monitor/lib/monitors.js

```diff
@@ -4,10 +4,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-'use strict'
+import createResponseTimeMiddleware from 'response-time'
+import metrics from './metrics.js'
 
-const createResponseTimeMiddleware = require('response-time')
-const { connectionsCount, connectionsTotal, responseTime } = require('./metrics')
+const { connectionsCount, connectionsTotal, responseTime } = metrics
```

Comment:
```suggestion
const {
  connectionsCount,
  connectionsTotal,
  responseTime
} = metrics
```
we need a linter for such cases :) (not for this PR of course)

Reply:
adapted in: 2ff3cf3f13d388cdfd56fd3634625730b950dc6c